feat(linter/plugins): implement `SourceCode#visitorKeys` property

overlookmotel opened the pull request:
Add support for obtaining visitor keys for AST via `context.sourceCode.visitorKeys`.

camc314 commented on apps/oxlint/src-js/plugins/source_code.ts:
```diff
@@ -118,7 +124,9 @@ export const SOURCE_CODE = Object.freeze({
   get visitorKeys(): { [key: string]: string[] } {
-    throw new Error('`sourceCode.visitorKeys` not implemented yet'); // TODO
+    // This is the path relative to `plugins.js` file in `dist` directory
+    if (visitorKeys === null) visitorKeys = require('./generated/visit/keys.js').default;
```
nit: could probably use the newer es syntax

```suggestion
    visitorKeys ??= require('./generated/visit/keys.js').default;
```

overlookmotel committed:
feat(linter/plugins): implement `SourceCode#visitorKeys` property (#14289)

Add support for obtaining visitor keys for AST via `context.sourceCode.visitorKeys`.
